pkgs/development/python-modules/maxminddb/default.nix

```diff
@@ -7,12 +7,13 @@
 }:
 
 buildPythonPackage rec {
-  version = "1.5.4";
+  version = "2.0.2";
   pname = "maxminddb";
+  disabled = pythonOlder "3.6";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "f4d28823d9ca23323d113dc7af8db2087aa4f657fafc64ff8f7a8afda871425b";
+    sha256 = "15z5557rn4yvrhnpdm9l4kczr151qv9px736hd361rlr2z98wpdr";
   };
 
   buildInputs = [ libmaxminddb ];
```

Comment:
missing maintainer

Reply:
I only noticed they were broken, I'm not going to maintain those packages.